integrated_cell/utils/utils.py

```diff
@@ -181,33 +188,64 @@ def w_init(x):
     optimizer = load_object(optim_name, kwargs_optim)
 
     if os.path.exists(save_path):
+        # if the save path exists
         if verbose:
             print("loading from {}".format(save_path))
         model_utils.load_state(network, optimizer, save_path, gpu_ids[0])
 
+    elif pretrained_path is not None:
+        # otherwise try to load from a pretrained path
+        if verbose:
+            print(
+                "loading pretrained {}.{} from {}".format(
+                    network_name, component_name, pretrained_path
+                )
+            )
```

Comment:
At some point, it would be cool to switch all these `print` statements to `logging.info` / `logging.warn`, etc.

Reply:
Not in the scope of this PR though.